### PR TITLE
RSpec parallel test additions

### PR DESCRIPTION
**Pull request by scruti**
## Changes in this PR:

This PR brings some additions for the usage of [parallel tests gem](https://github.com/grosser/parallel_tests) in our project:

- Sets up parallel tests to allow to run as a Spring binstub with `parallel_spec`.
- Adds env variable so parallel tests can run the full suite in the background while the developer can execute individual specs meanwhile.
- Setup parallel tests DBs creation as part of the devcontainer build process, allowing to run them after creation/rebuild.
- Add basic documentation on the docs.

### CODE REVIEW

File: bin/parallel_rspec [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env ruby
```
Comment (scruti) [inline bin/parallel_rspec:1]: Autogenerated file

File: bin/spring [hunk 2]
```diff
@@ -1,17 +1,14 @@
 #!/usr/bin/env ruby
```
Comment (scruti) [inline bin/spring:1]: File changes autogenerated when creating the new binstub.

File: README.md [hunk 3]
```diff
@@ -166,6 +166,33 @@ yarn run js:lint
...
 
+### RSpec parallel testing
+The service include [parallel_tests](https://github.com/grosser/parallel_tests) gem, that allows to split the RSpec test
+suite run accross multiple CPU cores.
+
+Each group will run against a separate database.
+To set it up:
+
+```
+RAILS_ENV=test rails parallel:create
```
Comment (starswan) [inline README.md:177]: RAILS_ENV=test isn't actually needed (but it doesn't hurt)

**Review by starswan — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (scruti) [reply README.md:177]: I had issues with `rails/rake parallel` setup commands running under `development` by default. Maybe something about the devcontainers though :shrug:

**Pull request merged**